src/content/en/2021/caching.md

```diff
@@ -17,4 +20,366 @@ featured_stat_3: TODO
+
+The most common Vary headers are: `accept-encoding`, `user-agent`, `origin`, `accept`.
+
+### TTL adoption
+
+Talk about 28% of mobile requests use neither Cache-Control nor Expires headers.
+
+The median HTML resource on mobile has a TTL of 14 days
+
+The median difference between resource TTL and resource age is -6 days, meaning the age is 6 days older than the TTL.
+
+## Cookie Cache adoption
+
+35.63% of Cacheable Desktop pages with set-cookie
+Mobile pages with set-cookie: 35.99%
+
+Another area of opportunity we discovered was with cacheable pages that are not using `set-cookie`. About 35% of Desktop and Mobile pages use `set-cookie` while over 64% of them do not.
+
+Large area of opportunity for pages that are actually cacheable, but are just missing this directive?
+
+Fun fact `test_cookie` is the most popular desktop and mobile cookie name at 4% of all cookies.
+
+{{ figure_markup(
+  caption="TODO",
+  content="4%",
+  classes="big-number",
+  sheets_gid="455647224",
+  sql_file="todo.sql"
+) }}
+
+The top cookies attributes used on Desktop and Mobile are SameSite and Secure. SameSite is on 53% of cookies set in Desktop responses and a whopping 68% on Mobile responses.
+
+Why is it a big deal that SameSite is at 68% on mobile?
+
+## What type of content are we caching?
+
+On one end of the spectrum, Font, CSS, and Audio files are over 99% cacheable.
+However, of well-used non-cacheable resources, HTML is the highest at 23.4% and Images follow closely with 10.1% being non-cacheable.
+
+When we consider the content that we cache by resource type, the most-cached resources are fonts with almost 100% of pages caching fonts. Followed quickly by CSS, audio, and video which are the most common static files on the web pages. However, the resources we cache the least due to their dynamic nature are text at 29.5% and HTML at 76.6%
+
+{{ figure_markup(
+  image="caching-by-resource-type.png",
+  caption="The percent of requests that use caching strategies by resource type.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=2036781114&format=interactive",
+  height=436,
+  sheets_gid="1202769738",
+  sql_file="non_cacheable_by_resource_type.sql"
+) }}
+
+When we compare the mobile data from 2020 and 2021, we notice a 5.1% increase in cacheable HTML. This tells us we may potentially be moving towards better using our CDNs to cache HTML pages that tend to change less frequently. Specifically, ones that are generated by Server-Side Rendered applications. Where usually with SSR when you visit a particular webpage if the content doesn't change, that url will serve the same HTML for potentially weeks or even months, making them highly cacheable.
+
+- Cacheable vs non-cacheable response
```

Comment:
When we look at the entirety of cacheable and non-cacheable resources, 

Let's take a look at how the TTL for the resources we are caching is distributed across all pages.
[fig]

When we take a look at the Median TTL across all resource types, we see that even if we are caching comparable amounts there is a much lengthier cache for mobile, particularly for HTML, audio and video.